error.go

```diff
@@ -115,6 +118,54 @@ const (
 	ErrorCodeInvalidSwipeData ErrorCode = "invalid_swipe_data"
 )
 
+// List of DeclineCode values.
+const (
+	DeclineCodeAuthenticationRequired         DeclineCode = "authentication_required"
+	DeclineCodeApproveWithID                  DeclineCode = "approve_with_id"
+	DeclineCodeCallIssuer                     DeclineCode = "call_issuer"
+	DeclineCodeCardNotSupported               DeclineCode = "card_not_supported"
+	DeclineCodeCardVelocityExceeded           DeclineCode = "card_velocity_exceeded"
+	DeclineCodeCurrencyNotSupported           DeclineCode = "currency_not_supported"
+	DeclineCodeDoNotHonor                     DeclineCode = "do_not_honor"
+	DeclineCodeDoNotTryAgain                  DeclineCode = "do_not_try_again"
+	DeclineCodeDuplicateTransaction           DeclineCode = "duplicate_transaction"
+	DeclineCodeExpiredCard                    DeclineCode = "expired_card"
+	DeclineCodeFraudulent                     DeclineCode = "fraudulent"
+	DeclineCodeGenericDecline                 DeclineCode = "generic_decline"
+	DeclineCodeIncorrectNumber                DeclineCode = "incorrect_number"
+	DeclineCodeIncorrectCVC                   DeclineCode = "incorrect_cvc"
+	DeclineCodeIncorrectPIN                   DeclineCode = "incorrect_pin"
+	DeclineCodeIncorrectZip                   DeclineCode = "incorrect_zip"
+	DeclineCodeInsufficientFunds              DeclineCode = "insufficient_funds"
+	DeclineCodeInvalidAccount                 DeclineCode = "invalid_account"
+	DeclineCodeInvalidAmount                  DeclineCode = "invalid_amount"
+	DeclineCodeInvalidCVC                     DeclineCode = "invalid_cvc"
+	DeclineCodeInvalidExpiryYear              DeclineCode = "invalid_expiry_year"
+	DeclineCodeInvalidNumber                  DeclineCode = "invalid_number"
+	DeclineCodeInvalidPIN                     DeclineCode = "invalid_pin"
+	DeclineCodeIssuerNotAvailable             DeclineCode = "issuer_not_available"
+	DeclineCodeLostCard                       DeclineCode = "lost_card"
+	DeclineCodeMerchantBlacklist              DeclineCode = "merchant_blacklist"
+	DeclineCodeNewAccountInformationAvailable DeclineCode = "new_account_information_available"
+	DeclineCodeNoActionTaken                  DeclineCode = "no_action_taken"
+	DeclineCodeNotPermitted                   DeclineCode = "not_permitted"
+	DeclineCodePickupCard                     DeclineCode = "pickup_card"
+	DeclineCodePinTryExceeded                 DeclineCode = "pin_try_exceeded"
```

Comment:
@daisy1754 Can you rename `Pin` to `PIN` to be consistent?

Reply:
@remi-stripe  done